packages/shorebird_cli/lib/src/commands/release/android_releaser.dart

```diff
@@ -96,10 +93,7 @@ Please comment and upvote ${link(uri: Uri.parse('https://github.com/shorebirdtec
 
     final File aab;
 
-    final publicKeyFile = argResults.file('public-key-path');
-    final base64PublicKey = publicKeyFile != null
-        ? base64Encode(publicKeyFile.readAsBytesSync())
-        : null;
+    final base64PublicKey = argResults.encodedPublicKey;
```

Comment:
Nice! I'd just inline this below in the call to buildAppBundle.

Reply:
In this instance I would rather keep the variable, since we might call a `buildApk` a couple of lines below, and we also pass the key there. With the variable we read the file just once.

Reply:
👍  makes sense!